Documentation/dt-bindings.md

```diff
@@ -247,3 +247,40 @@ new keycode to the actual key.
 		};
 	};
 ```
+
+### GPIO leds
+
+We also support the `gpio-leds` node. This node follows upstream binding except
+that most properties are not supported. Currently only `gpios` and
+`default-state` are supported in lk2nd.
+
+```
+gpio-leds {
+    compatible = "gpio-leds";
+
+    red {
+        gpios = <&tlmm 3 GPIO_ACTIVE_HIGH>;
+        default-state = "on";
+    };
+
+    green {
+        gpios = <&tlmm 4 GPIO_ACTIVE_HIGH>;
+        default-state = "keep";
+    };
+
```

Comment:
> should we use "keep" or "on"/"off"? I prefer not using "keep" for clarity.

I feel like "keep" makes no sense for lk1st and such a simple driver, since the only goal for us is to set initial state of the led (most often to "on"). I suggest just omitting it here and in the driver (will just be a warning anyway)

Reply:
It is a preperty supported by  linux gpio-driver. I think it's better to be a no-op without a warning. It's different with some other random definitely wrong things. But l think it's also okay to omit it anyway.